packages/nodes-base/nodes/CrowdDev/CrowdDevNode.node.ts

```diff
@@ -0,0 +1,31 @@
+import type { INodeType, INodeTypeDescription } from 'n8n-workflow';
+import { allProperties } from './descriptions';
+
+export class CrowdDevNode implements INodeType {
```

Comment:
Should not be "crowdDevNode" just "crowdDev". Is obviously the same for the name, defaults, context file, ...